docs/kyma/docs/05-01-overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+type: Configuration
+---
+
+## Default settings
+
+During the installation process, the Installer applies onto a cluster or Minikube all components defined in the `installer-cr-cluster.yaml.tpl` and `installer-cr.yaml.tpl` templates with their configuration defined in `values.yaml` files. It also imports the configuration settings defined in the `installer-config-cluster.yaml.tpl` and `installer-config-local.yaml.tpl` templates located under the `installation/resources` subfolder.
+
+> **NOTE:** The installation and configuration templates serve as the basis for creating the corresponding installation (`kyma-config-local.yaml` and `kyma-config-cluster.yaml`) and configuration (`kyma-installer-local.yaml` and `kyma-installer-cluster.yaml`) release artefacts.
+
+Both configuration files contain pre-defined overrides in Secrets and ConfigMaps that the Installer uses during the installation to replace default values specified in `values.yaml` files or to provide required configuration values. While the local configuration file contains hardcoded override values, the cluster configuration file is based on placeholders that are replaced with real values during the installation process.
+
+>**CAUTION:** Both `values.yaml` and the configuration files contain production settings that are tested and recommended. Note that you modify them on your own risk.
+
+## Installation configuration
+
+Before you start the Kyma installation process, you can customize the default settings in the installation and configuration files.
+
+### Components
+
+During the installation, the Installer creates the Kyma Installer image that contains all component charts located in the `kyma/resources` folder. Both cluster and local installation files contain a full list of these component names and their Namespaces.
+Components that are not an integral part of the default Kyma Lite package, are preceded with hashtags (#) which means that the Installer skips them during the installation process.   
+You can customize the component installation files by:
+- Removing hashtags in front of the component entries to enable a component installation
+- Adding hashtags to disable a component installation
+- Adding new components to the list along with their chart definition in the `kyma/resources` folder.
```

Comment:
here should also be a link to doc about building your own installer image, as this is needed when adding new chart

Reply:
Ok, I added it.